nexus_constructor/transformation_view.py

```diff
@@ -30,19 +30,18 @@ def _fill_in_existing_fields(self, current_vector):
         self.transformation_frame.x_spinbox.setValue(current_vector.x())
         self.transformation_frame.y_spinbox.setValue(current_vector.y())
         self.transformation_frame.z_spinbox.setValue(current_vector.z())
-        # update_function = find_field_type(self.transformation.dataset)
-        # if update_function is not None:
-        #     update_function(
-        #         self.transformation.dataset, self.transformation_frame.magnitude_widget
-        #     )
-        # if isinstance(self.transformation, NXLogTransformation):
-        #     self.transformation_frame.main_layout.addWidget(
-        #         QLabel(
-        #             "Transformation is an NXlog - currently these are not editable. "
-        #         )
-        #     )
-        #     self.transformation_frame.magnitude_widget.setVisible(False)
-        # Disabled whilst working on model change
+        update_function = find_field_type(self.transformation.value)
+        if update_function is not None:
+            update_function(
+                self.transformation.value, self.transformation_frame.magnitude_widget
+            )
+        if isinstance(self.transformation, NXLogTransformation):
```

Comment:
lets just remove this conditional, nxlogs are not going to work at the moment as we can't load in from NeXus files and there's no way to create them in the UI.